targets/lxde

```diff
@@ -11,6 +11,11 @@ CHROOTBIN='crouton-noroot startlxde'
 ### Append to prepare.sh:
 install lxde -- dictionaries-common chromium-browser network-manager wicd
 
+# xenial needs a few more packages.
+if release -ge xenial; then
+    install lxde-common lxsession-logout
```

Comment:
these are available on trusty and some debian releases (at least sid) as well.  Do we want to install them explicitly there as well?